src/modules/numbers/numbers.service.ts

```diff
@@ -64,4 +66,41 @@ export class NumbersService {
       requestingSystem: dbNumber.REQUESTING_SYSTEM,
     };
   }
+
+  /**
+   * Helper to sort new ids
+   */
+  sortIds(newUkefIds: UkefId[]): UkefId[] {
+    const sortedIds = [];
+    const newIdsByType = this.groupIdsByType(newUkefIds);
+    const sortednewIdsByType = this.sortGroupedByTypeIds(newIdsByType);
```

Comment:
Will `newIdsByType` ever be `null`, `undefined`? If yes we will need to add void check before any further execution.

Reply:
Hi,
Can't find a way how newIdsByType would be null or undefined. Input is validated and DB responses checked before this. So maybe check is not required.